mobile/native/src/lib.rs

```diff
@@ -36,4 +36,6 @@ pub use report_error::report_error_to_coordinator;
     unused_qualifications
 )]
 mod bridge_generated;
+mod hodl_invoice;
 mod position;
+mod unfunded_channel_opening_order;
```

Comment:
🙃 Hmmm, we should align the terminology between coordinator and app.

Reply:
what do you mean? the coordinator doesn't know of an unfunded channel opening order.

Reply:
Yeah, I think you are right. I was confused at first because this is implicitly about a single-funded channel, which is a different name.

Maybe it would be clearer with `unfunded_wallet_channel_opening_order`? The name would be annoyingly long though.